platform/linux-generic/m4/odp_dpdk.m4

```diff
@@ -2,12 +2,27 @@
 # Enable DPDK support
 ##########################################################################
 pktio_dpdk_support=no
+
+AC_ARG_ENABLE([pktio-dpdk],
+	      [AS_HELP_STRING([--enable-pktio-dpdk], [enable DPDK support for Packet I/O])],
```

Comment:
I would prefer making naming uniform with netmap pktio (e.g. '--enable-netmap' & '--enable-dpdk'). The 'pktio-' prefix can be dropped. This way if we choose to use other DPDK components later on (e.g. crypto) we don't have to rename the flag.